apis/dscinitialization/v1/serverless_types.go

```diff
@@ -0,0 +1,74 @@
+package v1
+
+import operatorv1 "github.com/openshift/api/operator/v1"
+
+type CertType string
+
+const (
+	SelfSigned CertType = "SelfSigned"
+	Provided   CertType = "Provided"
+)
+
+// ServerlessSpec configures KNative components used in Open Data Hub. Specifically,
+// KNative is used to enable single model serving (KServe).
+type ServerlessSpec struct {
+	// +kubebuilder:validation:Enum=Managed;Removed
+	// +kubebuilder:default=Removed
```

Comment:
https://docs.google.com/document/d/1A3la5Zg93RcvYk6HW08VKxHbtbEoDZNablEOUwahPmo/edit#heading=h.eb02e5dvc38p according to this doc, 
for clean install of operator, we will get kserve by default Managaed, 
so i think we should have Serverless and ServiceMesh by default set to Managed than Removed
FYI @israel-hdez @bartoszmajsak

Reply:
I'm not very in favor of enabling these components by default, since these can have an important impact on the cluster.

Yet, if the consensus tells that it would be safe, I think we can do it.

Reply:
@zdtsw I think default here refers to if nothing is set, it will be set to `Removed`. This is same as all other components. We can update our default DSCI CR to set these components to `Managed`